terraform/modules/lambda/variables.tf

```diff
@@ -49,6 +49,16 @@ variable "s3_document_bucket_arn" {
   type        = string
 }
 
+variable "secret_rails_api_user" {
+  description = "The Rails API user."
+  type        = string
+}
+
+variable "secret_rails_api_password" {
+  description = "The Rails API password."
```

Comment:
The use of "The" here and above seems inconsistent with the way other variables are described.